src/components/common/ProfilePanel.js

```diff
@@ -205,83 +204,47 @@ class ProfilePanel extends React.Component {
     }
   };
 
-  renderProfileButton = disabled => (
-    <div className={"panel-options-item" + (disabled ? "-disabled" : "")}>
-      {disabled && (
-        <img
-          style={{ position: "absolute", height: "60px", right: "-2px", zIndex: 20, opacity: 0.9 }}
-          alt="banner"
-          src={ComingSoonBanner}
-        />
-      )}
-      <span className={"panel-item-content"}>
-        <img className={"panel-item-icon"} alt="house" src={"img/house2.png"} />
-        <span className={"panel-item-name"}>Profile</span>
-      </span>
-    </div>
-  );
-
-  renderEditorButton = disabled => (
-    <div
-      key="editor"
-      className={"panel-options-item" + (disabled ? "-disabled" : "")}
+  renderEditorButton = () => (
+    <Button
+      className="panel-button"
+      key="editor-button"
+      size="lg"
+      block
       onClick={() => {
-        if (!disabled) {
-          this.setState({ redirectTo: "/editor" });
-        }
+        this.setState({ redirectTo: "/editor" });
       }}
     >
-      {disabled && (
-        <img
-          style={{ position: "absolute", height: "60px", right: "-2px", zIndex: 20, opacity: 0.9 }}
-          alt="banner"
-          src={ComingSoonBanner}
-        />
-      )}
-      <span className="panel-item-content">
-        <span className="panel-item-icon">
-          <img className={"panel-item-icon"} alt="house" src={House} />
-        </span>
-        <span className="panel-item-name">Editor</span>
-      </span>
-    </div>
+      <FontAwesomeIcon icon={faPencilAlt} />
+      <span> Editor</span>
```

Comment:
need nbsp here for consistency. I would suggest changing the Button class to use a flexbox and have the span with the text flex:1 to fill the rest of the space, but I know you hate the flex bois, so you don't have to :)

Reply:
Yup, you right! And naw I don't hate the flex bois, I'll try it out.